Sources/HttpPipeline/SharedMiddlewareTransformers.swift

```diff
@@ -16,7 +50,7 @@ public func basicAuth<A>(user: String, password: String)
           (
             writeStatus(.unauthorized)
               >>> writeHeader("WWW-Authenticate", "Basic")
```

Comment:
Was reminded that you can configure a "realm" to customize what the pop-up says. _E.g._:

```
WWW-Authenticate: Basic realm="My App"
```

So maybe we need to add the enum now?

``` .swift
writeHeader(.wwwAuthenticate(.basic(realm: ""))
// vs.
writeHeader(.wwwAuthenticate(.basic(realm: "My App"))
```

Reply:
Ah yeah, nice one, will do!